### PR TITLE
Add some table's description

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What is changed, added or deleted? (Required)

Add missing system tables to v6.5 docs

Note: The added content in this PR already exists in v7.5 and later versions. See [mysql Schema](https://docs.pingcap.com/zh/tidb/v7.5/mysql-schema#%E5%85%83%E6%95%B0%E6%8D%AE%E9%94%81%E7%9B%B8%E5%85%B3%E7%B3%BB%E7%BB%9F%E8%A1%A8) (zh), [mysql Schema](https://docs.pingcap.com/tidb/v7.5/mysql-schema#system-tables-related-to-metadata-locks) (en).

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lilin90 — COMMENTED** [review]
请确认一下，新增的内容适用于哪些 TiDB 版本？以便按需 cherry pick。

File: mysql-schema.md [hunk 1]
```diff
@@ -53,6 +53,17 @@ summary: 了解 TiDB 系统表。
...
+## 元数据锁相关系统表
+
+* `tidb_mdl_view` 元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info` TiDB 内部用于同步各节点的元数据锁的相关信息
+
+## DDL 相关系统表
+
+* `tidb_ddl_history` 记录了 DDL 语句的历史记录。
+* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL。
+* `tidb_ddl_reorg` TiDB 内部存放的正在执行的物理 DDL（例如加索引）的元数据，用于执行物理 DDL。
```
Comment (lilin90) [inline mysql-schema.md:65]: 与上下文保持格式一致
```suggestion
* `tidb_ddl_history` 记录了 DDL 语句的历史记录
* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL
* `tidb_ddl_reorg` TiDB 内部存放的正在执行的物理 DDL（例如加索引）的元数据，用于执行物理 DDL
```

File: mysql-schema.md [hunk 2]
```diff
@@ -53,6 +53,17 @@ summary: 了解 TiDB 系统表。
...
 
+## 元数据锁相关系统表
+
+* `tidb_mdl_view` 元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info` TiDB 内部用于同步各节点的元数据锁的相关信息
+
+## DDL 相关系统表
+
+* `tidb_ddl_history` 记录了 DDL 语句的历史记录
+* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL
```
Comment (lilin90) [inline mysql-schema.md:64]:
```suggestion
* `tidb_ddl_jobs` TiDB 内部存放的正在执行的 DDL 的元数据，用于执行 DDL
```

**Comment by lilin90** [comment]
> @wjhuang2016 新增的内容适用于 v7.1 吗？试用的话，请在 PR 描述里也勾选下 v7.1。

Confirmed with @wjhuang2016 that the changes also apply to v7.1. So I added the cherry pick label for v7.1.

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by tangenta — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**